blog.html

```diff
@@ -22,6 +22,125 @@
+                        </b><br>Finally, KIP-992 adds new timestamped-key and timestamped-range interactive queries for timestamped key-value state stores. This change improves the type safety of the IQv2 API. The existing RangeQuery now always returns only the value if issued against a timestamped key-value store.
+                    </li>
+                </ul>
+                <h3>Kafka Connect</h3>
+                <ul>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-976%3A+Cluster-wide+dynamic+log+adjustment+for+Kafka+Connect">KIP-976 Cluster-wide dynamic log adjustment for Kafka Connect</a>:
+                        </b><br>KIP-495 introduced the ability for users to dynamically change the log level of individual workers in their connect cluster. To address situations when it is not possible to target an individual worker, KIP-976 allows for dynamic log-level changes to be broadcast to all workers across the cluster in a single update.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-980%3A+Allow+creating+connectors+in+a+stopped+state">KIP-980 Allow creating connectors in a stopped state</a>:
+                        </b><br>This update allows users to create new Connectors in a STOPPED or PAUSED state, enabling use cases like migrating connectors. There is now a new optional "initial_state" field during connector creation.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-959%3A+Add+BooleanConverter+to+Kafka+Connect">KIP-959 Add BooleanConverter to Kafka Connect</a>:
+                        </b><br>KIP-959 adds a BooleanConverter to Connect, supporting serializing and deserializing the boolean primitive in a Kafka Connect schema. This implements both org.apache.kafka.connect.storage.Converter and org.apache.kafka.connect.storage.HeaderConverter.
+                    </li>
+                    <li><b><a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-970%3A+Deprecate+and+remove+Connect%27s+redundant+task+configurations+endpoint">KIP-970 Deprecate and remove Connect's redundant task configurations endpoint</a>:
+                        </b><br>Deprecates a redundant endpoint for fetching task configurations, eventually to be removed in Apache Kafka 4.0.
+                    </li>
+
+                </ul>
+                <h3>Summary</h3>
+                <p>Ready to get started with Apache Kafka 3.7.0? Check out all the details in the <a href="https://archive.apache.org/dist/kafka/3.7.0/RELEASE_NOTES.html">release notes</a> and <a href="https://kafka.apache.org/downloads">download</a> Apache Kafka 3.7.0.</p>
```

Comment:
Above you used `https://downloads.apache.org/kafka/3.7.0/RELEASE_NOTES.html` for the release notes.

Reply:
good point, I modified the previous one that also pointed to the archive too

Reply:
@stanislavkozlovski 3.6.x links should point to archive & 3.7.x links should point to download as per https://www.apache.org/legal/release-policy.html#when-to-archive

Reply:
TIL, thanks; fixed